### DISCUSSION TITLE
Show the field type within autocomplete suggestions

### DISCUSSION THREAD

**Discussion by JamesMDunn** [helix-editor/helix#10459] [General]
It would be awesome if somehow we could display the type of the fields within the autocomplete suggestions. For example, the following types here are 
```
accounts: string[]
data: string;
data8Hex: string;
isInner: boolean;
program: number;
```
![image](https://user-images.githubusercontent.com/43818256/192580788-02d84393-2136-4704-98a1-3e10e7c38ad3.png)

**Comment by EriKWDev** [comment]
I too would like this, though seeing if it's a module / function / field is also useful. Could perhaps switch between if we hold 'alt'?

**Comment by the-mikedavis** [comment]
This is up to the language server. `clangd` for example shows types in the docs for fields

![ls-type](https://user-images.githubusercontent.com/21230295/192652709-c6177b18-4a55-480e-ad74-2a6e95ac11f6.png)
